package exceptions;
import java.util.*;
public class ExceptionDemo {

	public static void main(String[] args) {
		/*
		 * // Example 1
		 * 
		 * System.out.println("Program is started");
		 * 
		 * Scanner sc = new Scanner(System.in);
		 * 
		 * int num = sc.nextInt();
		 * 
		 * System.out.println(100/num); // Arithmetic exception
		 * 
		 * int a[]=new int[5];
		 * 
		 * // Example 2
		 * 
		 * System.out.println("Enter the position where you want to add an number"); int
		 * num1 = sc.nextInt(); System.out.println("Enter the num to add in array"); int
		 * num2 = sc.nextInt();
		 * 
		 * a[num1] = num2; //Array index out of bound exception
		 * 
		 * System.out.println("Program is complted");
		 * System.out.println("Program is exiited");
		 */
		
		// Example 3
		/*
		String s="Welcome";
		int num3 = Integer.parseInt(s);  // Number format exception
		System.out.println(num3);
		*/
		
		//Example 4
		
		String name = null;
		System.out.println(name.length());
	}

}
